com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -580,6 +580,20 @@ private void OnEnable()
         /// <param name="prefab"></param>
         public void RemoveNetworkPrefab(GameObject prefab) => PrefabHandler.RemoveNetworkPrefab(prefab);
 
+        /// <summary>
+        /// Sets the maximum size of a single non-fragmented message (or message batch) passed through the transport.
+        /// This should represent the transport's MTU size, minus any transport-level overhead.
+        /// </summary>
+        /// <param name="size"></param>
+        public void SetMaxSingleMessageSize(int size) => MessageManager.SetMaxSingleMessageSize(size);
+
+        /// <summary>
+        /// Sets the maximum size of a message (or message batch) passed through the transport with the ReliableFragmented delivery.
+        /// Warning: setting this value too low may result in the SDK becoming non-functional with projects that have a large number of NetworkBehaviours or NetworkVariables, as the SDK relies on the transport's ability to fragment some messages when they grow beyond the MTU size.
+        /// </summary>
+        /// <param name="size"></param>
+        public void SetMaxFragmentedMessageSize(int size) => MessageManager.SetMaxFragmentedMessageSize(size);
+
```

Comment:
2 things:
1. I'd suggest `get; set;` property here (so that users could get the value without setting it)
2. I'd suggest naming these APIs as `MaximumTransmissionUnitSize` & `MaximumFragmentedMessageSize` because it looks like they boil down to those two concepts.